README.md

```diff
@@ -20,6 +20,6 @@ npm tst
 ## Local development
 
 1. Clone this repository
-1. Install the [Helix CLI](https://github.com/adobe/helix-cli): `npm install -g @adobe/helix-cli`
-1. Start Helix Pages Proxy: `hlx up` (opens your browser at `http://localhost:3000`)
+1. Install the [AEM CLI](https://github.com/adobe/helix-cli): `npm install -g @adobe/aem-cli`
```

Comment:
I know that this could be confusing but the project is still called `helix-cli` even if the npm package now is `aem-cli`

Reply:
I know but they did not rename the repo :D